src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -1001,8 +1003,9 @@ class VariantSampleMatrix[T](val hc: HailContext, val metadata: VariantMetadata,
 
   def deleteVA(path: List[String]): (Type, Deleter) = vaSignature.delete(path)
 
-  def downsampleVariants(keep: Long): VariantSampleMatrix[T] = {
-    sampleVariants(keep.toDouble / countVariants())
+  def downsampleVariants(keep: Double): VariantSampleMatrix[T] = {
```

Comment:
We had both downsampleVariants(keep) and sampleVariants(fraction).  If we just want fraction, just kill downsampleVariants?  (And make the python match.)  We could support both keep and fraction arguments in the interface.  I do like fraction because it doesn't require counting variants.